dll/directx/dsound_new/precomp.h

```diff
@@ -1,4 +1,5 @@
-#pragma once
+#ifndef PRECOMP_H
+#define PRECOMP_H
```

Comment:
Why is this change needed?
gcc should support `#pragma once` ?

Reply:
Yes, same as other compilers. But for some reason, at least RosBE Unix 2.1.2 can't compile it with `#pragma once`. I get an error "#pragma once in main file" and compilation doesn't go any further. :(
As far as I know, `#pragma once` works with more number of compilers than "#ifndef/#define/#endif", but is more farther from C/C++ standards.

Reply:
`#pragma once` doesn't work in precompiled header files.